runtime/lua/vim/provider/health.lua

```diff
@@ -426,12 +431,15 @@ end
 --- @param url string
 local function download(url)
```

Comment:
side note: could drop this and use `vim.net.request()` now ?

Reply:
Thought so too but didn't want to include it in this PR. I'll create a pr later and search through the codebase for custom `download` functions or calls to `curl`.